mas-foundation/src/main/java/sun/security/pkcs/PKCS9Attribute.java

```diff
@@ -743,7 +743,7 @@ public String getName() {
      * the name.
      */
     public static ObjectIdentifier getOID(String name) {
-        return NAME_OID_TABLE.get(name.toLowerCase());
+        return NAME_OID_TABLE.get(name.toLowerCase(Locale.ENGLISH));
```

Comment:
Except this change all other are formatting changes, can you please remove other changes